pom.xml

```diff
@@ -236,6 +236,21 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>io.opentracing.contrib</groupId>
+            <artifactId>opentracing-cassandra-driver</artifactId>
```

Comment:
Does this send spans for all Cassandra queries? If so I suggest removing them to reduce noise and may be wrap their entire operation in a single span (manually)

Reply:
I believe it's important to see how long queries are taking from distributed tracing perspective.

Reply:
I agree, this is why I suggested a single span that wraps all calls, otherwise we will end up with 100s if not 1000s of Cassandra queries that adds nothing but noise/sluggish experience.